forms/mainwindow.ui

```diff
@@ -355,7 +355,7 @@ border-bottom: 1px solid darkgrey;
                   </sizepolicy>
                  </property>
                  <property name="currentIndex">
-                  <number>0</number>
+                  <number>7</number>
```

Comment:
> Try and revert to the default values for stacks and indexes before committing, so I don't have to do it after the merge.

Did you know that those indexes don't affect the real Johnny. In MainWindow.cpp, there is code to reput the default one in the constructor. It only affects the way you see the .ui in qt designer. But since it seems important for you, I'll revert them each time I submit a PR.